From 99c84e66b7aca6fffa933907d68421c00e50dc78 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Sun, 22 Nov 2015 17:57:21 +0200
Subject: [PATCH 3/4] Make the configure script set the DESTDIR

---
 configure | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/configure b/configure
index 1a24852..222a540 100755
--- a/configure
+++ b/configure
@@ -1 +1,22 @@
 #!/bin/sh
+
+TEMP=`getopt -o '' --long enable-static,disable-shared,prefix:,host: \
+     -n 'configure' -- "$@"`
+
+if [ $? != 0 ] ; then exit 1 ; fi
+
+# Note the quotes around `$TEMP': they are essential!
+eval set -- "$TEMP"
+
+while true ; do
+    case "$1" in
+        --host) shift 2 ;;
+        --enable-static) shift ;;
+        --disable-shared) shift ;;
+        --prefix)
+            dir="$( echo $2 | sed 's_/_\\/_g' )"
+            sed -i "s/DESTDIR=.*/DESTDIR=${dir}/" makefile; shift 2 ;;
+        --) shift; break ;;
+        *) echo "Internal error!" ; exit 1 ;;
+    esac
+done
-- 
2.6.2

