From d9ae9cfeaf63fae6bc581b83712c98f106c46fe1 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Sun, 22 Nov 2015 17:57:21 +0200
Subject: [PATCH 3/4] Make the configure script set the DESTDIR

---
 configure | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/configure b/configure
index 1a24852..64ef63d 100755
--- a/configure
+++ b/configure
@@ -1 +1,17 @@
 #!/bin/sh
+
+TEMP=`getopt -o t:: --long target:: \
+     -n 'configure' -- "$@"`
+
+if [ $? != 0 ] ; then exit 1 ; fi
+
+# Note the quotes around `$TEMP': they are essential!
+eval set -- "$TEMP"
+
+while true ; do
+    case "$1" in
+        -t|--target) sed -i "s/DESTDIR=.*/DESTDIR=$2/" makefile ; shift 2 ;;
+        --) shift ; break ;;
+        *) echo "Internal error!" ; exit 1 ;;
+    esac
+done
-- 
2.6.2

