From c2a638126c9fcd7f35f6b82ff390e729e52ac4e1 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Sun, 22 Nov 2015 18:27:55 +0200
Subject: [PATCH 4/5] Add more options to the configure script

---
 configure | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/configure b/configure
index 64ef63d..0c970f2 100755
--- a/configure
+++ b/configure
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-TEMP=`getopt -o t:: --long target:: \
+TEMP=`getopt -o '' --long enable-static,disable-shared,prefix:,host: \
      -n 'configure' -- "$@"`
 
 if [ $? != 0 ] ; then exit 1 ; fi
@@ -10,8 +10,11 @@ eval set -- "$TEMP"
 
 while true ; do
     case "$1" in
-        -t|--target) sed -i "s/DESTDIR=.*/DESTDIR=$2/" makefile ; shift 2 ;;
-        --) shift ; break ;;
+        --host) shift 2 ;;
+        --enable-static) shift ;;
+        --disable-shared) shift ;;
+        --prefix) sed -i "s/DESTDIR=.*/DESTDIR=${2//\//\\\/}/" makefile; shift 2 ;;
+        --) shift; break ;;
         *) echo "Internal error!" ; exit 1 ;;
     esac
 done
-- 
2.6.2

