From 93a5ae2d676c5ac6aa7dc1bcf41426050c1e9fe7 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Thu, 9 Jun 2016 01:59:08 +0300
Subject: [PATCH 5/5] Produce a static binary

---
 makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makefile b/makefile
index 8201b3d..627da2d 100644
--- a/makefile
+++ b/makefile
@@ -15,7 +15,7 @@ $(OBJ)/%.o:$(SRC)/%.c
 	${CC} $(CFLAGS) -c $< -o $@
 
 jhead: $(objs) jhead.h
-	${CC} -o jhead $(objs) -lm
+	${CC} -o jhead $(objs) -lm -static
 
 clean:
 	rm -f $(objs) jhead
-- 
2.8.3

