From bdd59bb4dabe1ac23b004cf3fdb40a8b3e1c4aa2 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Sun, 22 Nov 2015 18:45:07 +0200
Subject: [PATCH 1/4] Add an install task

---
 Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index f4edc3c..dbaf850 100644
--- a/Makefile
+++ b/Makefile
@@ -12,6 +12,8 @@
 
 # macros --------------------------------------------------------------------
 
+DESTDIR=.
+
 CC = gcc
 LD = gcc
 RM = rm -f
@@ -89,3 +91,7 @@ $(PNGCRUSH)$(E): $(OBJS)
 
 clean:
 	$(RM) $(EXES) $(OBJS)
+
+install: $(PNGCRUSH)$(E)
+	mkdir -p ${DESTDIR}/bin/
+	cp $(PNGCRUSH)$(E) ${DESTDIR}/bin/
-- 
2.10.2

