From f5f4dbab2c81d9a3a6e908f9cc9d8c41c9188a71 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Sun, 22 Nov 2015 18:46:05 +0200
Subject: [PATCH 2/4] Add a configure script

---
 configure | 47 +++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 47 insertions(+)
 create mode 100755 configure

diff --git a/configure b/configure
new file mode 100755
index 0000000..6af6f2f
--- /dev/null
+++ b/configure
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+for arg in "$@"
+do
+    case "$arg" in
+    -- )
+        option="$arg"
+        ;;
+    * )
+        option="$arg"
+        ;;
+    esac
+    case "$arg" in
+    *=* )
+        optarg=`expr "X$arg" : 'X[^=]*=\(.*\)'`
+        ;;
+    * )
+        optarg=""
+        ;;
+    esac
+    case "$option" in
+    --help | -h )
+        echo "Usage:"
+        echo "    $0 [options]"
+        echo "Options:"
+        echo "    -h, --help              Show this help"
+        echo "    --prefix=PREFIX         Install in the specified prefix"
+        echo "                            [default: /usr/local]"
+        exit 0
+        ;;
+    --prefix=* )
+        prefix="$optarg"
+        shift
+        ;;
+    * )
+        echo "$0: warning: unknown option: $arg"
+        echo "Type \"$0 -help\" for help"
+        ;;
+    esac
+done
+
+sed_config="
+    s|DESTDIR=.*|DESTDIR=$prefix|g
+    s| *\$||
+"
+
+sed -i.bak "$sed_config" Makefile
-- 
2.8.3

