From a419719212782c73e114848064cb07d48bfebb50 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Mon, 13 Jun 2016 23:33:27 +0300
Subject: [PATCH 3/4] Use cc by default instead of gcc

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 0391a49..6f53ade 100644
--- a/Makefile
+++ b/Makefile
@@ -16,8 +16,8 @@
 
 DESTDIR=.
 
-CC = gcc
-LD = gcc
+CC = cc
+LD = cc
 RM = rm -f
 
 CPPFLAGS = -I.
@@ -35,7 +35,7 @@ TOOFAR_OK=
 #CFLAGS = -O2
 #CFLAGS = -O2 -fomit-frame-pointer -Wall
 #CFLAGS = -Os -fomit-frame-pointer -Wall
-CFLAGS = -O3 -funroll-loops -fomit-frame-pointer -Wall -Wshadow
+CFLAGS = -O3 -funroll-loops -fomit-frame-pointer
 # [note that -Wall is a gcc-specific compilation flag ("all warnings on")]
 
 LDFLAGS =
-- 
2.8.3

