From 2146fa99889f5005f2979bd48aefb0805333c442 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Thu, 9 Jun 2016 02:03:46 +0300
Subject: [PATCH 4/4] Produce a static binary

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6735f96..445e2d0 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ CPPFLAGS += -DNO_GZ
 # Cannot use this with libpng15 and later.
 # CPPFLAGS += -DINFLATE_ALLOW_INVALID_DISTANCE_TOOFAR_ARRR
 
-LDFLAGS =
+LDFLAGS = -static
 O = .o
 E =
 
-- 
2.10.2

