From 809b32f895286d1b6be4c941f6f56efcad77427f Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Thu, 9 Jun 2016 02:54:43 +0300
Subject: [PATCH 2/7] Add default LDFLAGS

---
 configure.bash | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.bash b/configure.bash
index cb4d3ad..25e293c 100755
--- a/configure.bash
+++ b/configure.bash
@@ -17,6 +17,7 @@ else
     COCOA_READER=0
     LCMS2=auto
 fi
+LDFLAGS="-lpng -static"
 EXTRA_CFLAGS=
 EXTRA_LDFLAGS=
 
-- 
2.12.0

