From b11934e2ae454175472ac6ab56aafeb443620d5b Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Tue, 17 May 2016 23:39:25 +0300
Subject: [PATCH 2/2] Honor mini_portile's lib and inc dirs

---
 configure.bash | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/configure.bash b/configure.bash
index 0cadcf0..2fa6c99 100755
--- a/configure.bash
+++ b/configure.bash
@@ -309,6 +309,13 @@ fi
 # pairs of possible *.h and lib*.so locations
 DIRS=()
 
+MINI_PORTILE_LIB_DIRS=($MINI_PORTILE_LIB_DIRS)
+MINI_PORTILE_INC_DIRS=($MINI_PORTILE_INC_DIRS)
+for((i=0; i < ${#MINI_PORTILE_LIB_DIRS[@]}; i+=1))
+do
+    DIRS+=("${MINI_PORTILE_INC_DIRS[$i]} ${MINI_PORTILE_LIB_DIRS[$i]}")
+done
+
 if command -v >/dev/null libpng-config; then
     DIRS+=("$(libpng-config --prefix) $(libpng-config --libdir)")
 fi
-- 
2.8.2

