From ff2200227fb23f17587bd45e6a2d421bf2fc2b0e Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Thu, 9 Jun 2016 04:45:34 +0300
Subject: [PATCH 6/7] Do not build static binary

---
 configure.bash | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.bash b/configure.bash
index ecf5128..5bb381d 100755
--- a/configure.bash
+++ b/configure.bash
@@ -17,7 +17,7 @@ else
     COCOA_READER=0
     LCMS2=auto
 fi
-LDFLAGS="-lpng -lz -llcms2 -lpthread -static"
+LDFLAGS="-lpng -lz -llcms2 -lpthread"
 EXTRA_CFLAGS=
 EXTRA_LDFLAGS=
 
-- 
2.12.0

