From ee573287eed74a6b5053f73b180f34cacc711cd9 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Thu, 9 Jun 2016 02:19:08 +0300
Subject: [PATCH] Link lm after ljpeg

---
 configure | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index c2ea37e..dfb3aa5 100755
--- a/configure
+++ b/configure
@@ -3381,7 +3381,7 @@ if ${ac_cv_lib_m_floor+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lm  $LIBS"
+LIBS="$LIBS -lm"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -3416,7 +3416,7 @@ if test "x$ac_cv_lib_m_floor" = xyes; then :
 #define HAVE_LIBM 1
 _ACEOF
 
-  LIBS="-lm $LIBS"
+  LIBS="$LIBS -lm"
 
 fi
 
-- 
2.8.3

