From 1f61d8d293bb134fc8534da6fc30f573c07612b3 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Fri, 17 Mar 2017 22:52:17 +0200
Subject: [PATCH 1/4] Add a configure script

---
 configure | 47 +++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 47 insertions(+)
 create mode 100755 configure

diff --git a/configure b/configure
new file mode 100755
index 0000000..4a36f72
--- /dev/null
+++ b/configure
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+for arg in "$@"
+do
+    case "$arg" in
+    -- )
+        option="$arg"
+        ;;
+    * )
+        option="$arg"
+        ;;
+    esac
+    case "$arg" in
+    *=* )
+        optarg=`expr "X$arg" : 'X[^=]*=\(.*\)'`
+        ;;
+    * )
+        optarg=""
+        ;;
+    esac
+    case "$option" in
+    --help | -h )
+        echo "Usage:"
+        echo "    $0 [options]"
+        echo "Options:"
+        echo "    -h, --help              Show this help"
+        echo "    --prefix=PREFIX         Install in the specified prefix"
+        echo "                            [default: /usr/local]"
+        exit 0
+        ;;
+    --prefix=* )
+        prefix="$optarg"
+        shift
+        ;;
+    * )
+        echo "$0: warning: unknown option: $arg"
+        echo "Type \"$0 -help\" for help"
+        ;;
+    esac
+done
+
+sed_config="
+    s|PREFIX = .*|PREFIX = $prefix|g
+    s| *\$||
+"
+
+sed -i.bak "$sed_config" guetzli.make
-- 
2.12.0

