From dd0e880ea7557bf6f59c398fd06ce1718a211789 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Fri, 17 Mar 2017 23:04:26 +0200
Subject: [PATCH 2/3] Add an install task to the Makefile

---
 Makefile     | 3 +++
 guetzli.make | 6 ++++++
 2 files changed, 9 insertions(+)

diff --git a/Makefile b/Makefile
index 52d56bd..4bb76b4 100644
--- a/Makefile
+++ b/Makefile
@@ -30,6 +30,9 @@ endif
 clean:
 	@${MAKE} --no-print-directory -C . -f guetzli.make clean
 
+install:
+	@${MAKE} --no-print-directory -C . -f guetzli.make install
+
 help:
 	@echo "Usage: make [config=name] [target]"
 	@echo ""
diff --git a/guetzli.make b/guetzli.make
index d4df6ad..8379164 100644
--- a/guetzli.make
+++ b/guetzli.make
@@ -26,6 +26,7 @@ ifeq ($(config),release)
   LDDEPS +=
   ALL_LDFLAGS += $(LDFLAGS) `pkg-config --libs libpng`
   LINKCMD = $(CXX) -o "$@" $(OBJECTS) $(RESOURCES) $(ALL_LDFLAGS) $(LIBS)
+  PREFIX = /usr/local
   define PREBUILDCMDS
   endef
   define PRELINKCMDS
@@ -53,6 +54,7 @@ ifeq ($(config),debug)
   LDDEPS +=
   ALL_LDFLAGS += $(LDFLAGS) `pkg-config --libs libpng`
   LINKCMD = $(CXX) -o "$@" $(OBJECTS) $(RESOURCES) $(ALL_LDFLAGS) $(LIBS)
+  PREFIX = /usr/local
   define PREBUILDCMDS
   endef
   define PRELINKCMDS
@@ -130,6 +132,10 @@ else
 	$(SILENT) if exist $(subst /,\\,$(OBJDIR)) rmdir /s /q $(subst /,\\,$(OBJDIR))
 endif
 
+install: $(TARGET)
+	mkdir -p $(PREFIX)/bin
+	cp $(TARGET) $(PREFIX)/bin/guetzli
+
 prebuild:
 	$(PREBUILDCMDS)
 
-- 
2.12.2

