From e185b168b536c9a4682e55e47cfa97280870ec1e Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Sat, 18 Mar 2017 00:31:45 +0200
Subject: [PATCH 3/4] Fix compilation on old OSX

---
 guetzli.make | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/guetzli.make b/guetzli.make
index 9b75e8e..46db32a 100644
--- a/guetzli.make
+++ b/guetzli.make
@@ -24,7 +24,7 @@ ifeq ($(config),release)
   ALL_RESFLAGS += $(RESFLAGS) $(DEFINES) $(INCLUDES)
   LIBS += -lpng -lgflags_nothreads -lz -lpthread
   LDDEPS +=
-  ALL_LDFLAGS += $(LDFLAGS) -s
+  ALL_LDFLAGS += $(LDFLAGS)
   LINKCMD = $(CXX) -o "$@" $(OBJECTS) $(RESOURCES) $(ALL_LDFLAGS) $(LIBS)
   PREFIX = /usr/local
   define PREBUILDCMDS
@@ -216,4 +216,4 @@ $(OBJDIR)/butteraugli.o: third_party/butteraugli/butteraugli/butteraugli.cc
 -include $(OBJECTS:%.o=%.d)
 ifneq (,$(PCH))
   -include $(OBJDIR)/$(notdir $(PCH)).d
-endif
\ No newline at end of file
+endif
-- 
2.12.0

