From 9745f0d30f72f47d39bc5aaa81fe76d4eb03da1c Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Thu, 13 Apr 2017 17:17:01 +0300
Subject: [PATCH 3/3] Produce a statically linked binary

---
 guetzli.make | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/guetzli.make b/guetzli.make
index 8379164..b7e998e 100644
--- a/guetzli.make
+++ b/guetzli.make
@@ -24,7 +24,7 @@ ifeq ($(config),release)
   ALL_RESFLAGS += $(RESFLAGS) $(DEFINES) $(INCLUDES)
   LIBS +=
   LDDEPS +=
-  ALL_LDFLAGS += $(LDFLAGS) `pkg-config --libs libpng`
+  ALL_LDFLAGS += $(LDFLAGS) `pkg-config --libs libpng` -lz -static
   LINKCMD = $(CXX) -o "$@" $(OBJECTS) $(RESOURCES) $(ALL_LDFLAGS) $(LIBS)
   PREFIX = /usr/local
   define PREBUILDCMDS
@@ -216,4 +216,4 @@ $(OBJDIR)/butteraugli.o: third_party/butteraugli/butteraugli/butteraugli.cc
 -include $(OBJECTS:%.o=%.d)
 ifneq (,$(PCH))
   -include $(OBJDIR)/$(notdir $(PCH)).d
-endif
\ No newline at end of file
+endif
-- 
2.12.2

